core/capabilities/ccip/configs/evm/contract_reader.go

```diff
@@ -271,6 +271,16 @@ var USDCReaderConfig = evmrelaytypes.ChainReaderConfig{
 				consts.EventNameCCTPMessageSent: {
 					ChainSpecificName: consts.EventNameCCTPMessageSent,
 					ReadType:          evmrelaytypes.Event,
+					EventDefinitions: &evmrelaytypes.EventDefinitions{
+						GenericDataWordDetails: map[string]evmrelaytypes.DataWordDetail{
+							consts.EventFilterCCIPMessageSent: {
+								Name: consts.EventFilterCCIPMessageSent,
```

Comment:
I would prefer the values that we are searching for to use different name then the name of the "entity". Maybe in this case this sounds fine, but this can set a bad example. We are searching for for eg. `MessageValue` that is contained within the `EventNameCCTPMessageSent` entity